### PR TITLE
AO3-6528 Collection participant error message reveals when users are suspended or banned

### PR DESCRIPTION
**Pull request by JonathanS223**
# Pull Request Checklist

* [X] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [X] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [X] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [X] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [X] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

[https://otwarchive.atlassian.net/browse/AO3-6528](https://otwarchive.atlassian.net/browse/AO3-6528)

## Purpose

This changes the wording of the response a user gets when they attempt to add someone to a challenge that has been banned. With this change, it is no longer possible to tell if the user selected had been banned. 

## Testing Instructions

    Ban or suspend a user
        Log in as admin
        Manage Users > Find Users 
        In “Name,” enter the name of the user you want to ban or suspend
        Find
        Follow the “Details” link
        Under “Warnings and suspensions,” choose the desired option to ban or suspend
        Fill in “Notes”
        Update
        Log out

    As the user who is not banned/suspended, create a collection and invite members to it
        Log in
        Hi, username! > My Collections > New Collection
        Fill in required info
        Submit
        Select the “Membership” option
        In “Add new members”, enter the name of the banned/suspended user
        Submit

## References

This is related to [AO3-6526](https://otwarchive.atlassian.net/browse/AO3-6526) but only obliquely as this error had the same issue but for co-creating.

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)? Ivedonestranger (he/him)

Jira Account: [Jonathan S. (ivedonestranger)](https://otwarchive.atlassian.net/jira/people/63c6d49f1d7734b550c190fd)

[AO3-6526]: https://otwarchive.atlassian.net/browse/AO3-6526?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/controllers/collection_participants_controller.rb [hunk 1]
```diff
@@ -105,7 +105,7 @@ def add
     if @participants_invited.empty? && @participants_added.empty?
       if pseud_results[:banned_pseuds].present?
         flash[:error] =
-            ts("%{name} is currently banned and cannot participate in challenges.",
+            ts("%{name} cannot participate in challenges.",
```
Comment (brianjaustin) [inline app/controllers/collection_participants_controller.rb:108]: Now that this is shorter, we can collapse it to 2 lines to please the Hound
```rb
flash[:error] = ts("%{name} cannot participate in challenges.",
                   name: pseud_results[:banned_pseuds].to_sentence)
```

Comment (JonathanS223) [reply app/controllers/collection_participants_controller.rb:108]: Thanks for that @brianjaustin ! I'll get that in when I push my next commit.

**Comment by JonathanS223** [comment]
Got slapped with covid and recovering. I'm still working on these changes and will work to get them updated.

**Review by brianjaustin — APPROVED** [review]
Thanks! Looks good to me!

**Pull request merged**